packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -1166,13 +1196,25 @@ class RawEditorState extends EditorState
     }
     // Snapshot the input before using `await`.
     // See https://github.com/flutter/flutter/issues/11427
-    final ClipboardData? data = await Clipboard.getData(Clipboard.kTextPlain);
-    if (data == null) {
+    final data = await widget.clipboardManager.getData();
+    if (data?.hasPlainText != true && data?.hasDelta != true) {
```

Comment:
Sorry, after thinking this through...
Could we have `if (data == null || data!.hasNoData)` ? (we can remove `hasPlainText`)
WDYT?